SwiftI18n/Classes/Main/I18nManager.swift

```diff
@@ -12,13 +12,27 @@ public class I18nManager {
 
     public static var instance = I18nManager()
     
+    public var overrideToFallbackLanguage = false
+    public var fallbackLanguage: String?
```

Comment:
Also this is not idiomatic Swift.
You should only have:
`public var fallbackLanguage: String?` - if it's null, then do nothing, if it's some then override.

With this you can have: overrideToFallbackLanguage: true & fallbackLanguage:nil | overrideToFallbackLanguage: false & fallbackLanguage:some which is not a valid state.

Reply:
I agree, my thought exactly, again task said explicitly to use these 2 members, my bad, should have optimised it during implementation

Reply:
In task, we didn't define that fallbackLanguage is optional. If we set it as optional value, then we can go only with one property and then we don' need bool property.

The main idea is to give possibility to user to fallback in production, while keeping keys (instead of missing translaiton) on staging builds so QA can easier check where we miss translation. E.g.:
```swift
#if STAGING
I18nManager.instance.fallbackLanguage = nil
#elseif APPSTORE
I18nManager.instance.fallbackLanguage = Language.eng.code
#endif
```

Reply:
We are using only one property now